src/dtclient/kubernetes_settings.go

```diff
@@ -257,3 +265,15 @@ func handleErrorArrayResponseFromAPI(response []byte, statusCode int) error {
 		return fmt.Errorf(sb.String())
 	}
 }
+
+func generateKubernetesMEIdentifier(kubeSystemUUID string) string {
```

Comment:
Would rather return an extra `error` instead of the implicit empty string error handling logic.